tests/yast2_gui/yast2_control_center.pm

```diff
@@ -124,10 +125,27 @@ sub start_kernel_settings {
 sub start_partitioner {
     search('partitioner');
     assert_and_click 'yast2_control-center-partitioner';
-    assert_screen 'yast2_control-center-partitioner_warning', timeout => 180;
-    send_key 'alt-y';
+    assert_screen [qw(yast2_control-center-partitioner_warning yast2_control-center-partitioner_expert)], 180;
+    # Define if storage-ng
+    set_var('STORAGE_NG', 1) if match_has_tag 'storage-ng';
```

Comment:
not sure ... I would prefer to keep decision maker as a human for this variable. It's too important to leave it like this

Reply:
We have the same approach in the installer, so for this test suite variable is not set. Of course we can also set it on all mediums, but this line of code doesn't hurt anyway.

Reply:
@asmorodskyi there is a long history behind that variable. Simply stated: We won't go back anytime soon :)